src/esm_runscripts/namelists.py

```diff
@@ -126,46 +126,77 @@ def nmls_remove(mconfig):
 
         namelist_changes = mconfig.get("namelist_changes", {})
         namelist_removes = []
+        error_message = (
+            "There is a syntax error, probably in your runscript, regarding namelist "
+            "changes (e.g. in a 'add_namelist_changes' block). "
+            "It seems that either 'namelist_file' or "
+            "'namelist_group' or 'both' are missing.\n"
+            "Please make sure that namelist changes are specified in the correct syntax (see example below)"
+            " and rerun your runscript.\n"
+            "Example for add_namelist_changes:\n"
+            "\t add_namelist_changes:\n"
+            "\t\t'<namelist_file>':\n"
+            "\t\t\t'<namelist_group>':\n"
+            "\t\t\t\t<variable>: <value>"
```

Comment:
The convention we are trying to impose is 4 spaces as indentation rule. Can you change that?